easybuild/tools/filetools.py

```diff
@@ -763,6 +763,18 @@ def apply_patch(patch_file, dest, fn=None, copy=False, level=None):
     apatch = os.path.abspath(patch_file)
     adest = os.path.abspath(dest)
 
+    if apatch.endswith(".gz"):
```

Comment:
I changed this for checking whether apatch does *not* end with ".patch". Thoughts ?